CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ changes.
 
 - `Greetings` message now contains also the hydra-node version.
 
+- New REST endpoint (`/protocol-parameters`) which provides protocol-parameters
+  used in the hydra-node.
```

Comment:
Nitpicky, but this is an HTTP path / endpoint. REST is more an overall paradigm.

```suggestion
- New HTTP endpoint (`GET /protocol-parameters`) which provides the current protocol parameters
  used by the hydra-node when validating transactions.
```